src/ports/postgres/modules/dbscan/dbscan.py_in

```diff
@@ -653,25 +651,26 @@ class DBSCAN_optimized(DBSCAN):
             )
             SELECT s.id,
                 ARRAY[i] AS coords,
-                CASE WHEN s.point[i] <= cutoff 
+                CASE WHEN s.point[i] < cutoff
```

Comment:
Do you think it makes sense to add a dev-check test with a small dataset to be sure we do not break this in future? I will leave it to you, we can skip it if you don't think it will help.

Reply:
Yes, that's a good idea--I'll add one